README.md

```diff
@@ -391,6 +391,30 @@ A function to start your application.  When invoked, `this` will be your instanc
 - `modulePackage`: the contents of the local module's package.json (if found)
 - `configFiles`: an object of filepaths for each found config file (filepath values will be null if not found)
 
+
+### prepareConfig(env)
+
+Requiring `env.configPath` and modules specified by CLI options or config files are executed here. This method gives your application a chance to change the `env` params before these requiring and to load `env.configFiles`.
+
+```js
+Hacker.prepareConfig = function(env) {
+  env.configProps = ['home', 'cwd'].map(function(dirname) {
+    return env.configFiles['.hacker'][name]
+  }).filter(function(filePath) {
+    return Boolean(filePath);
+  }).reduce(function(config, filePath) {
+    return mergeDeep(config, require(filePath));
+  }, {});
+  
+  if (env.configProps.hackerfile) {
+    env.configPath = path.resolve(env.configProps.hackerfile);
+    env.configBase = path.dirname(env.configPath);
+  }
+
+  Liftoff.prototype.prepareConfig.call(this, env);
```

Comment:
I don't like that we'd need to call this.  If this is just a normalization phase, it might be better to allow it in the configuration object given to Liftoff.